### PR TITLE
Add changelog for features that landed in cardano-node-8.7

### PR DESCRIPTION
**Pull request by lehins**
# Description

# Checklist

- [ ] Commit sequence broadly makes sense and commits have useful messages
- [ ] New tests are added if needed and existing tests are updated
- [x] When applicable, versions are updated in `.cabal` and `CHANGELOG.md` files according to the
      [versioning process](https://github.com/input-output-hk/cardano-ledger/blob/master/RELEASING.md#versioning-process).
- [ ] The version bounds in `.cabal` files for all affected packages are updated. **If you change the bounds in a cabal file, that package itself must have a version increase.** (See [RELEASING.md](https://github.com/input-output-hk/cardano-ledger/blob/master/RELEASING.md#versioning-process))
- [ ] All visible changes are prepended to the latest section of a `CHANGELOG.md` for the affected packages. **New section is never added with the code changes.** (See [RELEASING.md](https://github.com/input-output-hk/cardano-ledger/blob/master/RELEASING.md#changelogmd))
- [x] Code is formatted with [`fourmolu`](https://github.com/fourmolu/fourmolu) (use `scripts/fourmolize.sh`)
- [x] Cabal files are formatted (use `scripts/cabal-format.sh`)
- [x] [`hie.yaml`](https://github.com/input-output-hk/cardano-ledger/blob/master/hie.yaml) has been updated (use `scripts/gen-hie.sh`)
- [x] Self-reviewed the diff

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,16 @@ the `CHANGELOG.md` for each individual package for any changes relevant for deve
...
 
+## 8.7
+
+- Fix `PParamsUpdate` governance action ratification. Votes of DReps are now accounted for.
+- Move CDDL specification files from test packages into libraries that actually implement each era.
+- Add ability to retain Plutus logs for debugging when running scripts
+- Adition of `ConwayGovEvent`
+- Enforce no duplicates for submitted ProposalProcedures
+- Fix deserialization of `CostModels` in the `PParamsUpdate`. Invalid `CostModels` are no longer allowed, only `CostModels` for unrecognized Plutus versions are allowed starting with Conway
+- Improve deposit and refund calculation logic
```
Comment (teodanciu) [inline CHANGELOG.md:18]: Maybe the zero value?  Worth mentioning?

**Review by teodanciu — APPROVED** [review]
Looks good to me, just one suggestion about allowing zero coin in Value

File: CHANGELOG.md (hunk 1 above)
Comment (lehins) [reply CHANGELOG.md:18]: yeah, good point. Thanks

Comment (lehins) [reply CHANGELOG.md:18]: Done

**Pull request merged**